src/_assets/js/subscribe_form.js

```diff
@@ -0,0 +1,76 @@
+(function() {
+    // Form elements
+    const subscribeContainer = document.querySelector('.subscribe-container');
+    const subscribeForm = document.querySelector('.subscribe-container__form');
+    const errorMessage = document.querySelector('.error-message');
+    const successMessage = document.querySelector('.success-message');
+    const requiredFields = Array.from(subscribeForm.querySelectorAll("[required]"));
+    const formSubmitButton = document.querySelector('.form-submit-button');
+
+    // Add change eventListener to required fields
+    requiredFields.forEach(item => item.addEventListener('change', submitButtonHandler));
+
+    // Check request after clicking the submit button
+    formSubmitButton.addEventListener('click', () => checkRequest());
+
+    //  Create an XMLHttpRequest to get formbox response
+    function checkRequest() {
+        const url = 'https://services.google.com/fb/submissions/flutter-updates/';
+        const request = new XMLHttpRequest();
+
+        request.onreadystatechange = () => {
+            // readyState 4 means that the operation is complete.
+            if (request.readyState === 4) {
+                if (request.status >= 400) {
```

Comment:
Is better to check for a 200 status (which means success), any other status should be considered an error in this scenario.